odoo/addons/base/views/ir_module_views.xml

```diff
@@ -67,8 +67,8 @@
                             <field name="to_buy" invisible="1"/>
                             <field name="has_iap" invisible="1"/>
                             <button name="button_immediate_install" string="Activate" invisible="to_buy or state != 'uninstalled'" type="object" class="btn btn-primary me-1" groups="base.group_system"/>
-                            <a href="https://odoo.com/pricing?utm_source=db&amp;utm_medium=module#hosting=on_premise" target="_blank" class="btn btn-primary me-1"
-                               invisible="not to_buy or state not in ('uninstalled', 'uninstallable')" role="button">Upgrade</a>
+                            <a href="https://odoo.com/pricing?utm_source=db&amp;utm_medium=module#hosting=on_premise" target="_blank" class="btn btn-info me-1"
+                               invisible="not to_buy or state not in ('uninstalled', 'uninstallable')" role="button">Upgrade To Enterprise</a>
```

Comment:
@cth-odoo 
Ok if you say so can i apply it like this and do in v17.0 ?

Reply:
@duong77476-viindoo Yes, you can.

Reply:
@cth-odoo Thank you :rocket:

Reply:
@cth-odoo Can this be merged sir ?